src/Components/SearchList/SearchList.jsx

```diff
@@ -0,0 +1,22 @@
+import { useCallback } from 'react';
+import styles from './searchList.css';
+
+export default function SearchList({ items, onItemClick }) {
+  return items.map(item => <SearchItem key={item.id} item={item} onItemClick={onItemClick} />);
+}
+
+function SearchItem({ item, onItemClick }) {
+  const itemClickHandler = useCallback(
+    e => {
+      e.preventDefault();
```

Comment:
Do you really need preventDefault here ?
Please read about the method https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault